README.md

```diff
@@ -72,6 +72,7 @@ docker run -d \
 
 You should replace `<YOUR-ETH-NODE> `with your actual Ethereum node address.
 If you're using Infura, your Ethereum node address might look something like: `wss://mainnet.infura.io/ws/v3/your-infura-project-id`.
+Make sure you are using the websocket URL `ws`/`wss` and not the http URL `http`/`https`.
```

Comment:
@joshklop do we support wss? I haven't found parameter to pass ssl/tls certificate to net.Listener for websocket

Reply:
We don't support it for RPC. This is for connecting to the L1 node. We use geth's RPC client for that.

Reply:
Yes, we use websockets for L1 verification. I think @kirugan was asking if we support secure websockets `wss` for RPC.

Reply:
>  @kirugan was asking if we support secure websockets wss for RPC.

yes I thought we use it for RPC, thanks